<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRegister</class>
 <widget class="QDialog" name="frmRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>392</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="7" column="0">
           <widget class="QLabel" name="lblAddress">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="txtPassword2">
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QTextEdit" name="txtAddress"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblMail">
            <property name="text">
             <string>Mail:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="txtName"/>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="txtMail">
            <property name="inputMethodHints">
             <set>Qt::ImhEmailCharactersOnly</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblSurname">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblPhoneNumber">
            <property name="text">
             <string>Phone:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="txtSurname"/>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="txtPhoneNumber">
            <property name="inputMethodHints">
             <set>Qt::ImhPreferNumbers</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="txtUsername">
            <property name="inputMethodHints">
             <set>Qt::ImhLatinOnly|Qt::ImhLowercaseOnly|Qt::ImhNoAutoUppercase|Qt::ImhPreferLowercase</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="txtPassword">
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblUsername">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblPassword2">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QCheckBox" name="chkTermsOfService">
            <property name="text">
             <string>I have read the terms of use; I accept, declare and undertake.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="btnRegister">
            <property name="text">
             <string>Register</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtSurname</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>txtPassword2</tabstop>
  <tabstop>txtMail</tabstop>
  <tabstop>txtPhoneNumber</tabstop>
  <tabstop>txtAddress</tabstop>
  <tabstop>btnRegister</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmRegister</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
